20180224_kubernetes/index.html

```diff
@@ -0,0 +1,223 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=1024" />
+    <meta name="apple-mobile-web-app-capable" content="yes" />
+    <title>Introduction to Kubernetes | by Adrien Lemaire @Fandekasp</title>
+
+    <meta name="description" content="Kubernetes presentation on 2018/02/24 at OWDDM" />
+    <meta name="author" content="Adrien Lemaire" />
+
+    <link href="http://fonts.googleapis.com/css?family=Open+Sans:regular,semibold,italic,italicsemibold|PT+Sans:400,700,400italic,700italic|PT+Serif:400,700,400italic,700italic" rel="stylesheet" />
+    <link href="../impress.js/css/impress-demo.css" rel="stylesheet" />
+    <link href="kubernetes.css" rel="stylesheet" />
+</head>
+<body class="impress-not-supported">
+
+<div class="fallback-message">
+    <p>Your browser <b>doesn't support the features required</b> by impress.js, so you are presented with a simplified version of this presentation.</p>
+    <p>For the best experience please use the latest <b>Chrome</b>, <b>Safari</b> or <b>Firefox</b> browser.</p>
+</div>
+
+<div id="impress">
+
```

Comment:
There are slides missing here:

1. What are we going to learn about today and why is it important?
2. Who should know this?
3. Who are you, and why should we listen to **you**?